lib/utils.js

```diff
@@ -134,3 +136,29 @@ export function renderDevTools() {
     }
   }
 }
+
+export function deprecationNote() {
+  atom.notifications.addWarning("This feature will be deprecated soon!", {
+    description:
+      "Connecting to existing kernels via a `connection.json` file will be deprecated soon.\n\nFor some time now Hydrogen supports using [kernel gateways](https://nteract.gitbooks.io/hydrogen/docs/Usage/RemoteKernelConnection.html) for connection to existing kernels. Using that option is a lot simpler while being powerful.\n\nPlease get in touch with us if using remote kernels isn't a option for you.",
+    dismissable: true,
+    buttons: [
+      {
+        className: "icon icon-x",
+        text: "I really need this feature",
+        onDidClick: () => {
+          shell.openExternal("https://github.com/nteract/hydrogen/issues/new");
```

Comment:
Good point! I don't exspect a lot of people using this feature so I was a bit sloppy 😄